qemu/tests/cfg/x86_cpu_asyncpf.cfg

```diff
@@ -0,0 +1,9 @@
+- x86_cpu_asyncpf:
+    type = x86_cpu_asyncpf
+    only Linux
+    only q35
+    required_qemu = [6.0.0-20, )
+    start_vm = no
+    old_machine = 'pc-q35-rhel8.4.0'
```

Comment:
Fedora should has different name. Either support it or unsupported it, need to define it clearly.

Reply:
> Fedora should has different name. Either support it or unsupported it, need to define it clearly.

update and thanks.